### PR TITLE
Fix dynamic-as-bottom issue in Extent

### PR DESCRIPTION
**Pull request by srawlins**
This fixes an additional dynamic-as-bottom issue. I don't think it conflicts with #229 or #232.

I'll test internally to see if this breaks users.

### CODE REVIEW

**Comment by srawlins** [comment]
CC @psunkari

File: lib/charts/src/cartesian_area_impl.dart [hunk 1]
```diff
@@ -423,7 +423,8 @@ class DefaultCartesianAreaImpl implements CartesianArea {
     config.dimensions.take(dimensionAxesCount).forEach((int column) {
       var axis = _getDimensionAxis(column),
           sampleColumnSpec = data.columns.elementAt(column),
-          values = data.rows.map((row) => row.elementAt(column));
+          values = data.rows.map((row) => row.elementAt(column))
```
Comment (leafpetersen) [inline lib/charts/src/cartesian_area_impl.dart]: map<Comparable> instead?

Comment (srawlins) [reply lib/charts/src/cartesian_area_impl.dart]: I think you're type annotations got swallowed by GitHub Markdown :grinning: but I think I know what you mean.

Comment (leafpetersen) [reply lib/charts/src/cartesian_area_impl.dart]: updated. :)

Comment (srawlins) [reply lib/charts/src/cartesian_area_impl.dart]: Analyzer doesn't like that:

> error • The return type 'dynamic' isn't a 'Comparable', as defined by anonymous closure at lib/charts/src/cartesian_area_impl.dart:426:55 • return_of_invalid_type_from_closure

unless I use `as Comparable` inside as well.

Comment (leafpetersen) [reply lib/charts/src/cartesian_area_impl.dart]: Ah, implicit casts must be off, good!  Putting the cast inside of the lambda is better.  The cast on the outside probably will fail strong mode runtime checks.

Comment (srawlins) [reply lib/charts/src/cartesian_area_impl.dart]: Done!

**Pull request merged**